### PR TITLE
CSI-3093 update sidecars versions

### PR DESCRIPTION
**Pull request by zingero**
https://kubernetes-csi.github.io/docs/sidecar-containers.html#versioning

### CODE REVIEW

File: deploy/crds/csi.ibm.com_v1_ibmblockcsi_cr.yaml [hunk 1]
```diff
@@ -57,7 +57,7 @@ spec:
     imagePullPolicy: IfNotPresent
   - name: csi-provisioner
     repository: k8s.gcr.io/sig-storage/csi-provisioner
-    tag: "v2.1.0"
+    tag: "v2.2.0"
```
Comment (oriyarde) [inline deploy/crds/csi.ibm.com_v1_ibmblockcsi_cr.yaml:60]: what about snapshotter `v3.0.3`?

**Review by oriyarde — COMMENTED** [review]
I think it would be easier for you to first complete CSI-3092

File: deploy/crds/csi.ibm.com_v1_ibmblockcsi_cr.yaml (hunk 1 above)
Comment (zingero) [reply deploy/crds/csi.ibm.com_v1_ibmblockcsi_cr.yaml:60]: I thought we don't want to use beta versions, but done.

File: deploy/crds/csi.ibm.com_v1_ibmblockcsi_cr.yaml [hunk 2]
```diff
@@ -57,15 +57,15 @@ spec:
...
     imagePullPolicy: IfNotPresent
   - name: csi-attacher
     repository: k8s.gcr.io/sig-storage/csi-attacher
     tag: "v3.1.0"
     imagePullPolicy: IfNotPresent
   - name: csi-snapshotter
     repository: k8s.gcr.io/sig-storage/csi-snapshotter
-    tag: "v3.0.2"
+    tag: "v3.0.3"
```
Comment (ArbelNathan) [inline deploy/crds/csi.ibm.com_v1_ibmblockcsi_cr.yaml:68]: why not v4.0.0?

Comment (zingero) [reply deploy/crds/csi.ibm.com_v1_ibmblockcsi_cr.yaml:68]: Bumping to v4.x requires having an external snapshotter. We can't force it.

Comment (oriyarde) [reply deploy/crds/csi.ibm.com_v1_ibmblockcsi_cr.yaml:68]: we have a ticket for it (CSI-2873) when the time comes

**Review by ArbelNathan — APPROVED** [review]
LGTM

**Review by oriyarde — APPROVED** [review]
LGTM

**Pull request merged**